### ISSUE TITLE
Get 2018-Testbot (soon to be named)'s code ready and flashed.

### ISSUE THREAD

**Issue by wh1ter0se** [wh1ter0se/PowerUp-2018#82]
Also hoping to be finished with this by the end of the night. We need to update all talon values, ensure correct wiring, and calibrate pneumatic code. If all goes well, we're done with this by 8-8:30 tonight.

**Comment by wh1ter0se** [comment]
***[7:12 UPDATE]*** Dudding just bumped 2018-Testbot's ETA on functional electronics from Monday to Thursday. The code for it is ready enough, and calibration is being put on hold until it is ready.

**Comment by wh1ter0se** [comment]
Currently waiting on electrical to wire everything.
